### ISSUE TITLE
[SUGGESTION]: Randomized Hatching (and possibly randomized Mystery Eggs)

### ISSUE THREAD

**Issue by gammaflux594** [pokeclicker/pokeclicker#5406]
### Current Behavior

The game currently has multiple methods of sorting for which eggs you wish to hatch in the hatchery, and I think an option could possibly be added, or potentially be implemented as an additional challenge mode.

### Improved Behavior

The option to sort by random pokemon when you choose which pokemon to hatch in a hatchery, or alternatively implement it as a challenge mode where all pokemon you hatch are chosen at random. (This obviously excludes mystery eggs, fossils, and type eggs)

### Reason for change

Just for the challenge and because it's fun.

### Detailed Description

-Basically the ability to sort level 100 pokemon by random in the hatchery
-A new challenge mode where all eggs hatched are chosen completely at random, excluding special eggs like mystery, type, and fossils. All the level 100 pokemon in the hatchery would be replaced with a single button that has a "?" sprite.
